### ISSUE TITLE
Play2.6's AllowedHostsFilter prevents Authenticator Cookies from working

### ISSUE THREAD

**Issue by halfhp** [mohiva/play-silhouette#515]
Unfortunately I dont understand enough of what's going on to fully describe the issue here but so far as I can tell, the issue is that the `AllowedHostsFilter` removes cookies from the outgoing `Result` via it's `withHeaders` invocation, breaking authentication. Prior to 2.6 this filter was disabled by default.

A stop-gap fix is to disable all default filters:

`play.filters.enabled=[]`

**Comment by akkie** [comment]
@halfhp Have you tried to configure the filter?

Maybe with:
```
play.filters.hosts {
  allowed = ["localhost:9000"]
}
```

or 

```
play.filters.hosts {
  allowed = ["."]
}
```

**Comment by halfhp** [comment]
I have not but will give it a shot this evening.

**Comment by halfhp** [comment]
I tried both `play.filters.hosts` configurations above and unfortunately neither worked.

**Comment by akkie** [comment]
I've published a new version based on Play 2.6-RC2. Could you please test if the issue remains?

**Comment by halfhp** [comment]
Just tried  the latest snapsphot - no change :(

**Comment by halfhp** [comment]
hey @akkie it appears to be working now.  nice work whatever you did :-)

**Comment by akkie** [comment]
@halfhp It wasn't my merit. @sgoyal01 has fixed the issue with #517.

Thanks for testing :+1:
